fastlane/lib/fastlane/actions/install_xcode_plugin.rb

```diff
@@ -4,6 +4,12 @@ class InstallXcodePluginAction < Action
       def self.run(params)
         require 'fileutils'
 
+        if params.fetch(:github, ask: false)
+          github_api_url = params[:github].sub('https://github.com', 'https://api.github.com/repos')
+          release = self.fetch_json(github_api_url + '/releases/latest')
+          params[:url] = release['assets'][0]['browser_download_url']
```

Comment:
Showing a generic error when GH errors and a specific one for missing latest release now.